### ISSUE TITLE
Allow servers to disable worlddownloader

### ISSUE THREAD

**Issue by Smove** [dslake/WorldDownloader#64]
As some other plugins like rei's minimap & voxel minimap supports disabling them on some servers, WorldDownloader should allow the server to send a disable download state too.

The reason for this is that servers get copied by other players without any permissions to do that. Every server should have the decision to allow or disallow the downloading of the world.

Example:

Voxel Minimap can be disabled by sending the following message to players client:
p.sendMessage(new TextComponent("§3 §6 §3 §6 §3 §6 §e"));

A other way this can be done is by plugin message

**Comment by TealNerd** [comment]
since the source is open anyone could clone it, take out the bit that disables with the message, and use it freely.

**Comment by Lunatrius** [comment]
Don't do that. I've removed SBC messages in favor of a native plugin. SBC messages are completely _broken_ on Bukkit/Spigot/BungeeCord, they might break on vanilla/Forge eventually.

@TealNerd surely anyone can do that, assuming that they manage to download, compile and set up the mod properly - which is what, maybe 1% of the community? At least if filters out the people that don't know how to program/compile/etc.

**Comment by TealNerd** [comment]
Yeah, I guess that's true, but then you further distance the normal player from the knowledgeable one. For instance you can disable radar on your server, but then rather than everyone having radar, you leave it with 1% of people having radar and now 1% of players have an advantage over the remaining 99% rather than everyone being on even ground. This means almost 100% of the time it makes more sense to leave radar enabled as to not disadvantage a majority of your players. Of course a better solution would be a radar jamming mod but even most of those aren't very good at effectively blocking radar.

**Comment by Pokechu22** [comment]
FYI, I did implement a system like this (using plugin channels).  There is documentation [here](http://wiki.vg/User:Pokechu22/World_downloader).  It also has an integrated permission request system (which is a work in progress), to try and balance things out.

§-based systems are extremely rigid and also spam the chat; I'm avoiding them.

**Comment by Smove** [comment]
I don't like Chat(§) based systems too. Your solution ist great :+1:
